static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -267,7 +268,10 @@ const bootStandard = (): void => {
 
     newHeaderInit();
 
-    if (config.get('page.hasShowcaseMainElement')) {
+    const isLeftCol: boolean = isBreakpoint({ min: 'leftCol' });
```

Comment:
I think this is really saying we're **at least** leftCol, rather than `isLeftCol`, but this is a minor point. Do we need to re-evaluate this on resize, or are we not worried about that in Frontend? (I'm imagining orientation change being a common way this would come about, rather than people resizing their browsers on a computer)

Reply:
That's a fair comment, the variables doing what's intended but could be named more appropriately!

Regarding your second question, we should re-evaluate on resize - we're not and it does cause a bug, I actually raised another card to look at this https://trello.com/c/MofTHciu/433-adjust-secondary-column-position-on-viewport-breakpoint-changes as that fix may need a little more thought